trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/upload_reel/UploadReelScreen.kt

```diff
@@ -188,12 +184,19 @@ private fun UploadReelScreenTopBar(
         },
         title = stringResource(Res.string.new_trend),
         trailingContent = {
-            Chip(
-                text = stringResource(Res.string.page_number, 1, 3),
-                isSelected = false,
-                isEnabled = true,
-                onClick = {},
-            )
+            Box(
+                modifier = Modifier
+                    .clip(RoundedCornerShape(Theme.radius.full))
+                    .background(Theme.colorScheme.background.surfaceLow)
+                    .padding(horizontal = Theme.spacing._8, vertical = Theme.spacing._4),
+                contentAlignment = Alignment.Center,
+            ) {
+                Text(
+                    text = stringResource(Res.string.page_number, 1, 3),
+                    style = Theme.typography.label.small,
+                    color = Theme.colorScheme.shadePrimary
+                )
+            }
```

Comment:
same here what about make it a private coponent ?
```suggestion 
      trailingContent = { PageNumber() }
      
      @Composable
private fun PageNumber(){
    Box(
        modifier = Modifier
            .clip(RoundedCornerShape(Theme.radius.full))
            .background(Theme.colorScheme.background.surfaceLow)
            .padding(horizontal = Theme.spacing._8, vertical = Theme.spacing._4),
        contentAlignment = Alignment.Center,
    ) {
        Text(
            text = stringResource(Res.string.page_number, 1, 3),
            style = Theme.typography.label.small,
            color = Theme.colorScheme.shadePrimary
        )
    }
}

Reply:
got an idea to give the modifier to the text direct and it works
```
trailingContent = {
  Text(
      modifier = Modifier
          .clip(RoundedCornerShape(Theme.radius.full))
          .background(Theme.colorScheme.background.surfaceLow)
          .padding(horizontal = Theme.spacing._8, vertical = Theme.spacing._4),
      text = stringResource(Res.string.page_number, 1, 3),
      style = Theme.typography.label.small,
      color = Theme.colorScheme.shadePrimary,
      textAlign = TextAlign.Center
  )
}
```